internal/pkg/selinux/policy/selinux/services/udev.cil

```diff
@@ -20,6 +20,20 @@
 ; udevadm called by machined in its context
 (allow init_t udev_t (unix_stream_socket (connectto)))
 
+; Device subsystems, labeled by udev rules
+; RTC is important for security verification
+(type rtc_device_t)
+(call protected_device_f (rtc_device_t))
+; Could be exposed firmware storage
+(type mtd_device_t)
+(call protected_device_f (mtd_device_t))
+; Could reset the system
+(type wdt_device_t)
+(call protected_device_f (wdt_device_t))
+; TODO: should pods be able to access TPM?
```

Comment:
Should they? If so, no reason to label TPM specially if we are not going to use its label.